### PR TITLE
Subscription PR review fixes

### PR DESCRIPTION
**Pull request by maximburyak**

### CODE REVIEW

File: Documentation/4.0/Raven.Documentation.Pages/client-api/data-subscriptions/advanced-topics/subscription-with-revisioning.dotnet.markdown [hunk 1]
```diff
@@ -11,17 +11,43 @@ Such functionality allows keeping track of each change that was performed on a d
...
+In data subscription, Documents Revisions will be processed in pairs of subsequent entries.  
+Example: 
+Let us assume a user document that looks like:  
+
+`{  
+    Name:'James',  
+    Age:'21'  
+}`  
+
+We update the User document two times, in separate operations:
```
Comment (ayende) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/data-subscriptions/advanced-topics/subscription-with-revisioning.dotnet.markdown]: `two times` -> `twice`

File: Documentation/4.0/Raven.Documentation.Pages/client-api/data-subscriptions/advanced-topics/subscription-with-revisioning.dotnet.markdown [hunk 2]
```diff
@@ -11,17 +11,43 @@ Such functionality allows keeping track of each change that was performed on a d
...
+}`  
+
+We update the User document two times, in separate operations:  
+* We update the 'Age' field to the value of 22  
+* We update the 'Age' field to the value of 23  
+
+Data subscription's revision processing mechanism will receive pairs of revision in the following order:  
+
+
+| # | Previous | Current  |
```
Comment (ayende) [inline Documentation/4.0/Raven.Documentation.Pages/client-api/data-subscriptions/advanced-topics/subscription-with-revisioning.dotnet.markdown]: Better to include the full JSON here, not just the age

File: Documentation/4.0/Raven.Documentation.Pages/client-api/data-subscriptions/advanced-topics/subscription-with-revisioning.dotnet.markdown (hunk 1 above)
Comment (maximburyak) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/data-subscriptions/advanced-topics/subscription-with-revisioning.dotnet.markdown]: ok

File: Documentation/4.0/Raven.Documentation.Pages/client-api/data-subscriptions/advanced-topics/subscription-with-revisioning.dotnet.markdown (hunk 2 above)
Comment (maximburyak) [reply Documentation/4.0/Raven.Documentation.Pages/client-api/data-subscriptions/advanced-topics/subscription-with-revisioning.dotnet.markdown]: ok

**Pull request merged**